src/main/java/com/ververica/flink/table/gateway/operation/SelectOperation.java

```diff
@@ -218,13 +218,13 @@ public void cancelJob() {
 		}
 	}
 
-	private <C> ResultDescriptor executeQueryInternal(ExecutionContext<C> executionContext, String query) {
+	private ResultDescriptor executeQueryInternal(ExecutionContext executionContext, String query) {
 		// create table
 		final Table table = createTable(executionContext, executionContext.getTableEnvironment(), query);
 
 		boolean isChangelogResult = executionContext.getEnvironment().getExecution().inStreamingMode();
 		// initialize result
-		final Result<C, ?> result;
+		final Result<?> result;
```

Comment:
This `?` is not necessary either. The same in `ResultDescriptor.java`.

Reply:
not necessary, but this could tell the readers this class is a generic class